templates/README.md

```diff
@@ -43,10 +43,12 @@ If no nodes are assigned to the `footer` slot, the footer is hidden.
 
 **Note:** this template automatically includes `<header>`, `<main>`, `<aside>` and `<footer>` elements, so there's no need to include them inside the various slots.
 
+**Properties:**
+- `widthType` (string, default: `fullscreen`): The width that the template content will be constrained to. Accepts `fullscreen` or `normal`. Normal constrains width to `1230px`. 
```

Comment:
If the default is `fullscreen`, can you set the property to that in the component's constructor? Otherwise in JavaScript it'll be `undefined`.